ipaclient/plugins/cert.py

```diff
@@ -49,9 +49,14 @@ class CertRetrieveOverride(MethodOverride):
     )
 
     def forward(self, *args, **options):
-        certificate_out = options.pop('certificate_out', None)
-        if certificate_out is not None:
+        if 'certificate_out' in options:
+            certificate_out = options.pop('certificate_out')
+            if certificate_out is None:
+                raise errors.ValidationError(name='certificate-out',
+                                             error=_(u'cannot be empty'))
```

Comment:
Good point. I did not notice the same check is in the `ca` plugin as well so I unified these.